### DISCUSSION TITLE
I want the option to make the mouse wheel scroll horizontally

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#203] [Wants]
Let’s be honest, the web is largely vertical. That sucks. Sometimes you want to do something a bit different and you design something to scroll horizontally in your page and seems fine until you try to scroll the thing with a mouse and the only way is to awkwardly drag the scrollbar. Then you write some JavaScript and feel dirty inside ;-)

What if we could define the scroll direction for an element? Maybe something like this:

```css
div {
  scroll-orientation: horizontal;
}
```

<hr>

https://webwewant.fyi/wants/118/

**Comment by Psychpsyo** [comment]
Or you hold shift while scrolling.
That already makes it go sideways.

Not saying this is perfect, and being able to set the default scroll direction for an element to horizontal could be interesting but there's also mice that have a way to physically scroll sideways and this would be pretty annoying with those.

I feel like having browsers automatically do horizontal scrolling if the user starts scrolling in an element that can only scroll horizontally might make more sense.
